tests/services/validate_file_transfer_service/test_validate_file_transfer_service.py

```diff
@@ -0,0 +1,112 @@
+"""Module to test the validate file transfer service."""
+
+from pathlib import Path
+
+from cg.constants.constants import FileFormat
+from cg.services.validate_file_transfer_service.validate_file_transfer_service import (
+    ValidateFileTransferService,
+)
+
+
+def test_get_manifest_content(
+    validate_file_transfer_service: ValidateFileTransferService,
+    manifest_file: Path,
+):
+    """Test the get manifest file content method."""
+    # GIVEN a manifest file
+
+    # WHEN getting the content of the manifest file
+    manifest_content = validate_file_transfer_service.get_manifest_file_content(
+        manifest_file=manifest_file, manifest_file_format=FileFormat.TXT
+    )
+
+    # THEN assert that the content is a list
+    assert isinstance(manifest_content, list)
```

Comment:
maybe also check that the list has the correct content, or at least strings

Reply:
I think at that point we are testing the writer. Here I just want to test that the file I want ot access is being read.